kfs-core/src/main/resources/edu/arizona/kfs/fp/document/validation/configuration/DistributionOfIncomeAndExpenseValidation.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+ Copyright 2010 The Kuali Foundation
+ 
+ Licensed under the Educational Community License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.opensource.org/licenses/ecl2.php
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<beans xmlns="http://www.springframework.org/schema/beans"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.springframework.org/schema/beans
+        http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+
+  <!-- Had to include a bean in orig. parent bean because it had no list to merge with -->
+  <bean id="DistributionOfIncomeAndExpense-addAccountingLineValidation" parent="DistributionOfIncomeAndExpense-addAccountingLineValidation-parentBean" scope="prototype">
+    <property name="validations">
+      <list>
+        <bean parent="AccountingDocument-AddAccountingLine-DefaultValidation" scope="prototype"/>
+        <bean parent="AccountingDocument-globalTransactionAccountingLineValuesAllowedValidation" scope="prototype">
+          <property name="accountingDocumentParameterPropertyName" value="document"/>
+          <property name="accountingLineParameterPropertyName" value="accountingLine"/>
+        </bean>
+      </list>
+    </property>
+  </bean>
+
+  <!-- Had to include a bean in orig. parent bean because it had no list to merge with -->
+  <bean id="DistributionOfIncomeAndExpense-updateAccountingLineValidation" parent="DistributionOfIncomeAndExpense-updateAccountingLineValidation-parentBean" scope="prototype">
+    <property name="validations">
+      <list>
+        <bean parent="AccountingDocument-UpdateAccountingLine-DefaultValidation" scope="prototype"/>

```

Comment:
Same thing as for the add validation, where this differs than in KFS3, and the kuali parent matches what I have here. So is it a change between versions, or is the KFS6 kuali parent incorrect?